### ISSUE TITLE
 cors settings

### ISSUE THREAD

**Issue by leolabdev** [Alt-Org/Altzone-Server#253]
please return cors settings for our localhost 
![image](https://github.com/user-attachments/assets/1ce71318-89bd-43bb-a137-80cca467e8a0)

**Comment by MikhailDeriabin** [comment]
@leolabdev 
Please describe your problem using the provided issue template or at least describe it in details

**Comment by leolabdev** [comment]
The screenshot highlights the issue, but it can't be classified as a bug. It simply looks like the CORS setting for our localhost:5173 was removed, which is causing the problem with loading image galleries.

**Comment by MikhailDeriabin** [comment]
The screenshot does not provide any information except for the error code. If it is not a bug, it is a feature request. 

I can get the data from the url you just provided, please describe your problem in details

**Comment by leolabdev** [comment]
Feature: Enable CORS for localhost:5173 in Reverse Proxy Settings

**Comment by leolabdev** [comment]
To reproduce this error. You can run our project in dev mode: https://github.com/Alt-Org/Altzone-WebPages/wiki/Getting-Started and go through the navbar to the gallery/pictures or gallery/comics pages.

**Comment by leolabdev** [comment]
We are also experiencing the same issue with the dev subdomain: https://dev.altzone.fi/en/picture-galleries. We discussed this earlier, but at that time, you were unable to reproduce it. Last time, it worked in some browsers and not in others, but now the issue appears in all of my browsers."

**Comment by leolabdev** [comment]
> We are also experiencing the same issue with the dev subdomain: https://dev.altzone.fi/en/picture-galleries. We discussed this earlier, but at that time, you were unable to reproduce it. Last time, it worked in some browsers and not in others, but now the issue appears in all of my browsers."

Remark: It works on iPhone...

**Comment by leolabdev** [comment]
Hi, yes, thank you, everything worked for the localhost, but the problem is still present on the dev.

I don't know what the joke is, but on some devices it works, on some it doesn't. In my case, it only works on an iPhone.

I'm attaching a screenshot, and I'll also conduct an additional survey with other team members to see if it works for them.
![image](https://github.com/user-attachments/assets/ed78eee2-17e9-40ae-a3e7-365b14e210a3)

**Comment by Jonroi** [comment]
i have same problems with pc but in my android phone it works perfectly *dev.altzone*
<img width="1273" alt="image" src="https://github.com/user-attachments/assets/54bc6d57-dd1c-4e7e-92d8-cc3171ddd8e9">

**Comment by leolabdev** [comment]
@kaisahakola 
 ![image](https://github.com/user-attachments/assets/78f70c40-1509-4470-be3e-73a68c68d4fb)

**Comment by MikhailDeriabin** [comment]
@leolabdev 

ok, provide me info about request, response and console logs:

- screenshots of request and response headers from _network_ tab of `artGalleries`
- all console errors as a text

**Comment by leolabdev** [comment]
@MikhailDeriabin 

Say if you need something more. Btw, It started working for me in some browsers...

![image](https://github.com/user-attachments/assets/44606084-5c53-46b2-bd71-fc7556a1c998)


Errors: "
[Intervention] Images loaded lazily and replaced with placeholders. Load events are deferred. See https://go.microsoft.com/fwlink/?linkid=2048113
picture-galleries:1  Access to fetch at 'https://altzone.fi/api/public/site/artGalleries' from origin 'https://dev.altzone.fi' has been blocked by CORS policy: No 'Access-Control-Allow-Origin' header is present on the requested resource. If an opaque response serves your needs, set the request's mode to 'no-cors' to fetch the resource with CORS disabled.
altzone.fi/api/public/site/artGalleries:1 
        
        
        Failed to load resource: net::ERR_FAILED
2472-dbcb0b4d00d30cb6.js:1  Object
(anonymous) @ 2472-dbcb0b4d00d30cb6.js:1
picture-galleries:1  The resource https://dev.altzone.fi/_next/static/css/1a4cbd52ca857d76.css was preloaded using link preload but not used within a few seconds from the window's load event. Please make sure it has an appropriate `as` value and it is preloaded intentionally.
picture-galleries:1  The resource https://dev.altzone.fi/_next/static/css/ea0e3f48e6ea3223.css was preloaded using link preload but not used within a few seconds from the window's load event. Please make sure it has an appropriate `as` value and it is preloaded intentionally.
picture-galleries:1  The resource https://dev.altzone.fi/_next/static/css/43f58c00b3c2118c.css was preloaded using link preload but not used within a few seconds from the window's load event. Please make sure it has an appropriate `as` value and it is preloaded intentionally.
picture-galleries:1  The resource https://dev.altzone.fi/_next/static/css/1ce9c5969a1d918f.css was preloaded using link preload but not used within a few seconds from the window's load event. Please make sure it has an appropriate `as` value and it is preloaded inte
"

**Comment by MikhailDeriabin** [comment]
@leolabdev 

I need the response headers tab also

**Comment by leolabdev** [comment]
@MikhailDeriabin  

It is empty
![image](https://github.com/user-attachments/assets/a220f0d2-0f82-42c6-874c-034a86266479)

**Comment by MikhailDeriabin** [comment]
@leolabdev 

I meant _response headers_, it is above _request headers_ in _headers_ tab

**Comment by MikhailDeriabin** [comment]
@leolabdev 

I am closing the issue. As it was discussed on our meeting the problem with localhost is resolved and partially with dev.altzone as well. 

We decided that at some point we could try to rename the folder artGalleries to art-galleries, as this could cause a problems in some browsers.
